deno/fhirchat-demo/postcss.config.ts

```diff
@@ -8,6 +9,7 @@ import customMediaPlugin from "npm:postcss-custom-media";
 export const config = {
   plugins: [
     postcss(),
+    postcssSass(),
```

Comment:
have you tried without `postcss()` & _only_ `postcssSass()`?

Reply:
it works with `postcssSass()` as well and I can remove the `postcss()`. I had to use `postcss()` at first because I was getting a strange error related to `postcssSass()` being improperly imported. Since then it's been fixed.